CODEOWNERS

```diff
@@ -1056,8 +1056,8 @@ build.json @home-assistant/supervisor
 /homeassistant/components/seven_segments/ @fabaff
 /homeassistant/components/sfr_box/ @epenet
 /tests/components/sfr_box/ @epenet
-/homeassistant/components/sharkiq/ @JeffResc @funkybunch @AritroSaha10
-/tests/components/sharkiq/ @JeffResc @funkybunch @AritroSaha10
+/homeassistant/components/sharkiq/ @JeffResc @funkybunch
+/tests/components/sharkiq/ @JeffResc @funkybunch
```

Comment:
Can you please move this change to a separate PR, with corresponding motivation?